lib/apiml.js

```diff
@@ -82,6 +82,19 @@ function ApimlConnector({ hostName, port, discoveryUrls,
                           discoveryPort, tlsOptions, eurekaOverrides, isClientAttls }) {
   Object.assign(this, { hostName, port, discoveryUrls,
                         discoveryPort, tlsOptions, eurekaOverrides, isClientAttls });
+  //TODO config should never be checked through env var, but is temporarily needed to temporarily read gateway's ATTLS state to provide it with Eureka info it can work with.
+  const clientGlobalAttls = process.env['ZWE_zowe_network_client_tls_attls'] == 'true';
+  const clientGatewayAttls = process.env['ZWE_components_gateway_zowe_network_client_tls_attls'] == 'true';
+  const clientAttls = clientGlobalAttls || clientGatewayAttls;
+  this.isGatewayClientAttls = false;
+  if ((clientGlobalAttls !== false) && (clientLocalAttls !== false) && (!clientAttls)) {
```

Comment:
I can't see the `clientLocalAttls` definition is it a typo and should be `clientGatewayAttls`? 
In that case i do not quite understand this condition, i believe it will never pass as clientGlobalAttls and clientGatewayAttls both are defined booleans. if both are true then clientAttls will always be true.

Reply:
you're right, this was a mistake of me copying over from the util.js function.
it's not going to have the same logic.